components/dashboard/src/settings/Preferences.tsx

```diff
@@ -98,6 +98,17 @@ export default function Preferences() {
             </div>
             {useDesktopIde && <>
                 <div className="mt-4 space-x-4 flex">
+                    <SelectableCard className="w-36 h-40" title="VS Code Desktop" selected={defaultDesktopIde === 'code-desktop'} onClick={() => actuallySetDefaultDesktopIde('code-desktop')}>
+                        <div className="flex justify-center mt-3">
+                            <img className="w-16 filter-grayscale self-center" src={vscode} />
+                        </div>
+                    </SelectableCard>
+                    <SelectableCard className="w-36 h-40" title="VS Code Desktop" selected={defaultDesktopIde === 'code-desktop-insiders'} onClick={() => actuallySetDefaultDesktopIde('code-desktop-insiders')}>
+                        <div className="flex justify-center mt-3">
+                            <img className="w-16 filter-grayscale self-center" src={vscode} />
```

Comment:
**question**: I've asked this a couple of times[[1](https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933884412)][[2](https://github.com/gitpod-io/gitpod/pull/5024#pullrequestreview-724349137)][[3](https://gitpod.slack.com/archives/C01KGM9DVRC/p1632842972139700?thread_ts=1632840549.138400&cid=C01KGM9DVRC)] but still not entirely sure about the answer. Does it make sense to use the `VS Code Insiders` icon[[4](https://commons.wikimedia.org/wiki/File:Visual_Studio_Code_Insiders_1.36_icon.svg)] here?

Reply:
@gtsiolis Does the logo still look different after the gray filter has been applied? It only differs in color, right?

Reply:
See also the colored versions for reference. You can find these in [this early design draft](https://www.figma.com/file/HsOzavksL3WgRkxPwnzhs8/Add-local-IDEs-options-to-the-user-settings-%235641?node-id=1001%3A77).

| VS Code | VS Code Insiders |
|-|-|
| ![VS Code](https://user-images.githubusercontent.com/120486/142206981-475baefe-829c-4b43-a06f-2101c76c29e8.png) | ![VS Code Insiders](https://user-images.githubusercontent.com/120486/142206985-edbbb6c4-cb7b-458b-8529-3573d41b3936.png) |